MobiMart/customer/views.py

```diff
@@ -9,6 +9,31 @@
 from .models import *
 
 # Create your views here.
+def checkpass(str):
```

Comment:
And don't use str if possible, use already a keyword so try using different variable name.